ui/widgets/calendar.js

```diff
@@ -150,7 +151,14 @@ return $.widget( "ui.calendar", {
 		}
 
 		if ( this._needsRefresh() ) {
-			this._refresh();
+			if ( this.options.numberOfMonths > 1 && this.date.year() === this.viewDate.year() ) {
+				this.viewDate.adjust( "M", this.options.numberOfMonths *
+					( this.date.month() > this.viewDate.month() ? 1 : -1 )
+				);
+				this.refresh();
+			} else {
+				this._refresh();
```

Comment:
Can we rename `_refresh` and drop the `refresh()` call from it? Then any caller that currently calls `_refresh()` will have to call both methods. That seems like a good tradeoff.